packages/devtools/lib/src/timeline/timeline.css

```diff
@@ -48,7 +48,13 @@
     overflow: hidden;
 }
 
-.flame-chart .flame-chart-item {
+.flame-chart-section {
+    min-width: 100%;
+    position: relative;
```

Comment:
nit: can this be absolute instead of relative?

Reply:
relative allows the two sections to be stacked vertically. Does absolute positioning yield better performance?

Reply:
relative without an offset is a little odd. to just stack vertically you shouldn't need relative but doesn't really matter. shouldn't impact performance significantly. The general idea is if we know exactly how large the flame chart sections should be we might as well position them manually to avoid unexpected issues or flashes of content at different sizes if a frame happens to render before all the bars we plan to layout are laid out.

Reply:
Got it. Switched to absolute. 👍